enctests/testframework/main.py

```diff
@@ -407,10 +475,17 @@ def idiff_compare(source_clip, test_ref, testname, comparisontestinfo, source_pa
     if cmdresult != 0:
         result['result'] = "Unable to extract file for test"
     else:
-        cmd = apptemplate.format(originalfile=sourcepng, newfile=distortedpng, idiff_bin=IDIFF_BIN, newfilediff=diffpng)
+        cmd = apptemplate.format(originalfile=sourcepng, 
+                                 newfile=distortedpng.as_posix(),
+                                idiff_bin=IDIFF_BIN, 
+                                newfilediff=diffpng.as_posix())
         print(f"\n\nIdiff command: {cmd}", file=log_file_object)
         
-        output = subprocess.run(shlex.split(cmd), check=False, stdout=subprocess.PIPE).stdout
+        try:
+            output = subprocess.run(shlex.split(cmd), check=False, stdout=subprocess.PIPE).stdout
+        except FileNotFoundError as e:
+            print(f"ERROR: File not found, when executing {cmd}, check that the executable exists.")
+            exit(1)
```

Comment:
Do we really want to exit here?

Reply:
If the testing framework is unable to find the application for extracting a frame, we are in pretty bad shape, so I feel an exit is deserved.

Reply:
Fair enough. Just thought since we have several ways of comparing results we could let it continue and throw a warning. I'm fine with exiting too.